src/db/clients/mysql.js

```diff
@@ -135,6 +136,18 @@ export function wrapQuery(item) {
   return `\`${item}\``;
 }
 
+export const getSchema = async (connection) => {
```

Comment:
Since we are not using the function anywhere else. I think is better keep it private (The same suggestion for the other DB clients).